src/ImageSharp/MetaData/Profiles/Exif/ExifWriter.cs

```diff
@@ -86,14 +85,27 @@ public byte[] GetData()
                 return null;
```

Comment:
im not sure about this one, i think this was supposed to return null, if only the Exif Id code is present. Is that correct?

Reply:
@dlemstra What do you think?